conda.recipe/meta.yaml

```
@@ -3,21 +3,22 @@
 {% set license = data.get('license') %}
 {% set description = data.get('description') %}
 {% set url = data.get('url') %}
-# this will reproduce the version from tags to match versioningit
-{% set raw_version = environ.get('GIT_DESCRIBE_TAG', '0.0.0') %}
-{% set version = raw_version.lstrip('v') %}
-{% set version_number = environ.get('GIT_DESCRIBE_NUMBER', '0') | string %}
+# this will get the version set by environment variable
+{% set version = environ.get('VERSION') %}
+{% set version_number = version.split('+')[0] %}
+# change the build number by hand if you want to rebuild the package
+{% set build_number = 0 %}
 
 package:
   name: "quicknxs"
-  version: {{ version }}
+  version: {{ version_number }}
 
 source:
   path: ..
 
 build:
   noarch: python
-  version: {{ version_number }}
+  version: {{ build_number }}
```

The version is inherited from the package
```suggestion
  build: {{ build_number }}
```